<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollBar" name="scrMin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="ScrMax">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollBar" name="scrKernel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Kernel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblKernel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Kernel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMax">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brnOK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brnApply">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
